src/server/svr_jobfunc.c

```diff
@@ -2719,7 +2719,7 @@ Time4resvFinish(struct work_task *ptask)
 				newreq->rq_perm |= ATR_DFLAG_MGWR;
 				strcpy(newreq->rq_user, pbs_current_user);
 				strcpy(newreq->rq_host, server_host);
-				strcpy(newreq->rq_ind.rq_delete.rq_objname, presv->ri_qs.ri_resvID);
+				strcpy(newreq->rq_ind.rq_manager.rq_objname, presv->ri_qs.ri_resvID);
```

Comment:
why did you change rq_delete to rq_manager?

Reply:
The data structure used to store the information for ResvOccurEnd request is rq_manager, it would be ideal to use rq_manager for better code readability.